DotNetDBF/DBFHeader.cs

```diff
@@ -20,26 +20,20 @@
 
 namespace DotNetDBF
 {
-
-    static public class DBFSigniture
-    {
-        public const byte NotSet = 0,
-                          WithMemo = 0x80,
-                          DBase3 = 0x03,
-                          DBase3WithMemo = DBase3 | WithMemo;
-
-    }
-
-    [Flags]
-    public enum MemoFlags : byte
+    public enum DBFSignature : byte

```

Comment:
This is better but is going to be a binary breaking change, need to up version number.